project/Dependencies.scala

```diff
@@ -41,7 +41,7 @@ object Dependencies {
     "com.typesafe.scala-logging"    %% "scala-logging"       % "3.9.5",
 
     "org.parboiled" % "parboiled-core" % "1.4.1",
-    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-e0584dbdc")
+    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-902d77f8-SNAP")
```

Comment:
I think I already asked this, but just to be clear will ask again.

So the plan is that you merge the Rawls PR first and then update the version string here so it isn't a SNAP version?

Reply:
Ah, I saw your previous response.